src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,78 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORM = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom.trim(), DATE_FORM);
+        LocalDate toDate = LocalDate.parse(dateTo.trim(), DATE_FORM);
+
+        StringBuilder report = new StringBuilder();
+        report.append("Report for period ")
+                .append(fromDate.format(DATE_FORM))
+                .append(" - ")
+                .append(toDate.format(DATE_FORM))
+                .append("\n");
+
+        Map<String, Integer> totalEarnedMap = new HashMap<>();
```

Comment:
[Don’t use two-dimension arrays and HashMap(or any other Map).](https://github.com/mate-academy/jv-salary-info/blob/master/checklist.md#dont-use-two-dimension-arrays-and-hashmapor-any-other-map)